File: benchmarks/accuracy/fixtures/javascript/control_fixed_url_request.js
const express = require('express');

const router = express.Router();

router.get('/search', async (req, res) => {
    const term = req.query.q;
    const resp = await fetch('https://api.example.com/search', { method: 'POST', body: term });
    res.send(await resp.text());
});

module.exports = router;

File: benchmarks/accuracy/fixtures/go/control_fixed_url_request.go
package main

import (
	"io"
	"net/http"
	"strings"
)

func search(w http.ResponseWriter, r *http.Request) {
	term := r.URL.Query().Get("q")
	resp, err := http.Post("https://api.example.com/search", "text/plain", strings.NewReader(term))
	if err != nil {
		http.Error(w, "fetch failed", http.StatusBadGateway)
		return
	}
	defer resp.Body.Close()
	io.Copy(w, resp.Body)
}
